radio/src/audio.h

```diff
@@ -156,31 +158,36 @@ struct AudioFragment {
   uint8_t type;
   uint8_t id;
   uint8_t repeat;
+  int8_t _volume;
```

Comment:
Any particular reason for using `_volume` instead of `volume`?

Reply:
This was meant to make distinguishing the existing volume from the overriding volume. Finally _volume gets calculated into volume.